packages/node-cli/__test__/commands/account/setup/SetupAction.test.js

```diff
@@ -0,0 +1,38 @@
+jest.mock('../../../../src/commands/base/BaseAction');
+jest.mock('../../../../src/SdkExecutor');
+jest.mock('../../../../src/services/NodeTranslationService');
+
+const SdkExecutor = require('../../../../src/SdkExecutor');
+const NodeTranslationService = require('../../../../src/services/NodeTranslationService');
+
+const sdkExecutorExecuteMock = jest.spyOn(SdkExecutor.prototype, 'execute');
+const nodeTranslationServiceGetMessageMock = jest.spyOn(NodeTranslationService, 'getMessage');
+
+const AuthenticateActionResult = require('../../../../src/services/actionresult/AuthenticateActionResult');
+const SetupAction = require('../../../../src/commands/account/setup/SetupAction');
+
+const AUTH_MODE = {
+	OAUTH: 'OAUTH',
+	REUSE: 'REUSE',
+};
+
+describe('SetupAction execute(params)', () => {
+	it('should fail with exeption message generated in the java-core', async () => {
```

Comment:
I would rename the test "should fail when exception message is returned from java-core" or something like this

Reply:
Description changed.